apps/cowswap-frontend/src/common/pure/NewModal/index.cosmos.tsx

```diff
@@ -0,0 +1,85 @@
+import ICON_ARROW from 'assets/icon/arrow.svg'
+import SVG from 'react-inlinesvg'
+import styled from 'styled-components/macro'
+
+import { IMAGE_ACCOUNT, MOCK_TOKEN } from 'common/constants/cosmos'
+import { UI } from 'common/constants/theme'
+import { IconSpinner } from 'common/pure/IconSpinner'
+import { Stepper } from 'common/pure/Stepper'
+
+import { NewModal, NewModalContentBottom, NewModalContentTop } from './index'
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+`
+
+const ArrowRight = styled(SVG)`
+  --size: 12px;
+  width: var(--size);
+  height: var(--size);
+  margin: auto;
+
+  > path {
+    fill: var(${UI.COLOR_TEXT2});
+  }
+`
+
+const ModalFixtures = {
+  'new modal + content top/bottom': (
```

Comment:
I would think fixtures should be more simple, just as a show case with minimal code.

Also, its a bit hard to see the modal boundaries currently since the bg color in the fixture is the same as the modal:

<img width="1791" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/e1e13675-0215-4642-99c7-a8a6314ff9b9">

Reply:
Is this the component you wanted to have a BACK button instead of a close?